src/helpers/columnDefinitions.ts

```diff
@@ -18,10 +18,29 @@ export type StudentsTableRowData = StudentWithMetadata & {
 
 export const defaultStudentsColDef: ColDef = {
   flex: 1,
-  editable: true
+  editable: true,
+  cellStyle: (params) => {
```

Comment:
it's better to use cell class params or smth like that. but not mandatory to change